### ISSUE TITLE
No topic to know which devices have been found

### ISSUE THREAD

**Issue by jac459** [eschava/broadlink-mqtt#90]
There should be a topic that should send the list of devices found. So far, the only way I found is by reading the logs.
Am I missing something ?

**Comment by eschava** [comment]
You are correct, log is the only source to see all discovered devices if any
Do you want to have a special MQTT topic to see all discovered devices there?

**Comment by jac459** [comment]
Oh it would be awesome, please.

**Comment by jac459** [comment]
Oh yes, absolutely, it would be awesome.
.
